python/lsst/pipe/base/connections.py

```diff
@@ -919,6 +919,56 @@ def adjustQuantum(self, inputs, outputs, label, data_id):
                 )
         return {}, {}
 
+    def getSpatialBoundsConnections(self) -> Iterable[str]:
+        """Return the names of regular input and output connections whose data
```

Comment:
I just realized here and below `name` is (unfortunately) overloaded, in that we have the name of the class variable (which I think is intended) and also the `name` property of the connection.  I'm not sure how we distinguish this in our documentation now?